validation/process_capabilities_fail/process_capabilities_fail.go

```diff
@@ -20,7 +20,7 @@ func main() {
 	if err != nil {
 		util.Fatal(err)
 	}
-	g.AddProcessCapabilityBounding("CAP_TEST")
+	g.Config.Process.Capabilities.Bounding = append(g.Config.Process.Capabilities.Bounding, "CAP_TEST")
```

Comment:
Is there a bug in the helper function that we should fix?

Reply:
no, I think it is fine that the helper function checks whether the input is valid and rejects unknown capabilities